app/services/keyword_translator.rb

```diff
@@ -45,7 +45,7 @@ def translations
   end
 
   def translated_keyword
-    return keyword if translations.nil?
+    return keyword if translations.nil? || translations.length == 0
```

Comment:
We should use the Rails `blank?` method which automatically checks for either nil or empty array in this case.
So, `return keyword if translations.blank?` should do it.